addons/context_actions/functions/fnc_teleportZeus.sqf

```diff
@@ -17,4 +17,10 @@
 
 params ["_posASL"];
 
-player setPosASL _posASL;
+curatorMouseOver params ["_type", "_entity"];
+
+if (_type isEqualTo "OBJECT") then {
```

Comment:
Right now when teleport action is used when hovered over any object nothing will happen if it is not a vehicle. I think it would be better to check here for a vehicle instead of checking that inside `teleportIntoVehicle`.

Reply:
Agreed. 👍